### PR TITLE
Handle constexpr getelementptr

### PR DESCRIPTION
**Pull request by akokoshn**

### CODE REVIEW

**Comment by akokoshn** [comment]
CI: https://github.com/NilFoundation/zkLLVM/pull/345

File: include/nil/blueprint/parser.hpp [hunk 1]
```diff
@@ -666,12 +664,28 @@ namespace nil {
...
                     case llvm::Instruction::PtrToInt:
                         handle_ptrtoint(expr, expr->getOperand(0), frame);
                         break;
+                    case llvm::Instruction::GetElementPtr: {
+                        std::vector<int> gep_indices;
+                        for (unsigned i = 2; i < expr->getNumOperands(); ++i) {
+                            int gep_index = resolve_number<int>(frame, expr->getOperand(i));
+                            gep_indices.push_back(gep_index);
+                        }
+                        auto gep_res = handle_gep(expr->getOperand(0), expr->getOperand(1), expr->getOperand(0)->getType(), gep_indices, frame);
```
Comment (makxenov) [inline include/nil/blueprint/parser.hpp:684]: `expr->getOperand(0)->getType()` gives wrong type that is not suitable to `handle_gep`

**Review by makxenov — CHANGES_REQUESTED** [review]

File: include/nil/blueprint/parser.hpp (hunk 1 above)
Comment (akokoshn) [reply include/nil/blueprint/parser.hpp:684]: good catch, thanks!

Comment (akokoshn) [reply include/nil/blueprint/parser.hpp]: done

**Review by makxenov — APPROVED** [review]

**Pull request merged**